python/hail/api1/tests.py

```diff
@@ -907,3 +907,10 @@ def test_pca(self):
         _, _, loadings = (hc.balding_nichols_model(3, 100, 100)
                           .pca('g.GT.nNonRefAlleles()', k=2))
         self.assertEqual(loadings, None)
+
+    def test_split_multi_hts(self):
+        vds = hc.import_vcf("src/test/resources/sample.vcf")
+        vds = vds.annotate_genotypes_expr("g = {GT: g.GT}")
+        self.assertRaisesRegexp(FatalError,
+                                "split_multi_hts: genotype_schema must be the HTS genotype schema",
+                                vds.split_multi_hts)
```

Comment:
missing parens after split_multi_hts

Reply:
Thats the correct signature for `assertRaisesRegexp`

Reply:
ya, same as `lambda: vds.split_multi_hts()`

Reply:
Oh my mistake - I thought I saw one above that was different, but I'm just losing my mind. Carry on